index.html

```diff
@@ -31,8 +33,26 @@ <h2 class="underline-animation">About us</h2>
             <div class="menu-toggle"></div>
         </div>
     </header>
-    <div id="bg"></div>
-    
+    <div class="main-container">
+        <div id="bg"></div>
+        <div class="welcome-container">
+            <h2>Welcome</h2>
+            <h3>To</h3>
+            <h1>GitHub</h1>
```

Comment:
I'd use an h2 instead of an h1 and then style it as an h1

Reply:
Why though? I don't remember anything about not using it.

Reply:
He said something about the fact that every H1 should be different in every page, but that was for google stuff so I don't know if it matters now